e2etest/pkg/frr/container/container.go

```diff
@@ -42,50 +101,38 @@ func StartContainer(containerName string, testDirName string) error {
 }
 
 // Returns the IPv4 and IPv6 addresses of the container.
```

Comment:
maybe we should update the comment as this function no longer returns the ips but setting them

Reply:
done